include/utility/vpd_specific_utility.hpp

```diff
@@ -466,5 +466,88 @@ inline bool findCcinInVpd(const nlohmann::json& i_JsonObject,
     logging::logMessage("VPD type not supported. Can't find CCIN");
     return false;
 }
+
+/**
+ * @brief API to reset data of the FRU on DBus
+ *
+ * This API resets the data for particular interfaces of a FRU on DBus.
+ *
+ * @param[in] i_objectPath - DBus object path of the FRU.
+ * @param[in] io_interfaceMap - Interface and its properties map.
+ */
+inline void resetDataOnDbus(const std::string& i_objectPath,
+                            types::InterfaceMap& io_interfaceMap)
+{
+    try
+    {
+        std::array<const char*, 0> l_interfaces;
+        const types::MapperGetObject& l_getObjectMap =
+            dbusUtility::getObjectMap(i_objectPath, l_interfaces);
+
+        const std::vector<std::string>& l_vpdRelatedInterfaces{
+            constants::operationalStatusInf, constants::inventoryItemInf,
+            constants::assetInf};
+
+        for (const auto& [l_service, l_interfaceList] : l_getObjectMap)
+        {
+            if (l_service.compare(constants::pimServiceName) !=
+                constants::STR_CMP_SUCCESS)
+            {
+                continue;
+            }
+
+            for (const auto& l_interface : l_interfaceList)
+            {
+                if ((l_interface.find("com.ibm.ipzvpd") != std::string::npos) ||
```

Comment:
can we use constants::ipzVpdInf as it is already defined.

Reply:
Done